### ISSUE TITLE
Add JEI support

### ISSUE THREAD

**Issue by josephcsible** [josephcsible/PistonCompression#1]
Compression recipes should be visible in JEI.

**Comment by ghost** [comment]
Yes please! This feature would make the mod vastly more approachable for new users!

**Comment by Deshiba** [comment]
The documentation for the JEI is at the [JEI github wiki](https://github.com/mezz/JustEnoughItems/wiki)
With some helpfull [examples](https://github.com/mezz/JustEnoughItems/wiki/List-of-Plugin-Implementations) for implementation
